docs/about/abbreviations.rst

```diff
@@ -0,0 +1,87 @@
+Abbreviations
+=============
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Abbreviation
+     - Meaning
+   * - ARD
+     - Analysis Ready Data
+   * - ASF
+     - Alaska Satellite Facility
+   * - CARD
+     - CEOS Analysis Ready Data
+   * - CARD4L
+     - CEOS Analysis Ready Data for Land
+   * - CEOS
+     - Committee on Earth Observation Satellites
+   * - COG
+     - Cloud Optimized GeoTIFF
+   * - CRS
+     - Coordinate Reference System
+   * - DEM
+     - Digital Elevation Model
+   * - DSM
+     - Digital Surface Model
+   * - EGM
+     - Earth Gravitational Model
+   * - EPSG
+     - European Petroleum Survey Group
+   * - ESA
+     - European Space Agency
+   * - ETAD
+     - Extended Timing Annotation Dataset
+   * - EW
+     - Extra Wide Swath Mode
+   * - GETASSE
+     - Global Earth Topography And Sea Surface Elevation
+   * - GRD
+     - Ground Range Detected
+   * - ISLR
+     - Integrated Side Lobe Ratio
+   * - IW
+     - Interferometric Wide Swath Mode
+   * - KML
+     - Keyhole Markup Language
+   * - MGRS
+     - Military Grid Reference System
+   * - NESZ
+     - Noise Equivalent Sigma Zero
+   * - NRB
+     - Normalised Radar Backscatter
+   * - NRT
+     - Near Real Time
+   * - OGC
+     - Open Geospatial Consortium
+   * - ORB
+     - Ocean Radar Backscatter
+   * - OSV
+     - Orbit State Vector
+   * - PSLR
+     - Peak Side Lobe Ratio
+   * - RTC
+     - Radiometric Terrain Correction
+   * - SAR
+     - Synthetic Aperture Radar
+   * - SLC
+     - Single Look Complex
+   * - SM
+     - Stripmap Mode
+   * - SNAP
+     - Sentinel Application Platform
+   * - STAC
+     - SpatioTemporal Asset Catalog
+   * - UTM
+     - Universal Transverse Mercator
+   * - VRT
+     - Virtual Raster Format
```

Comment:
Virtual Raster Tile

Reply:
Ah yeah, I'm blaming Copilot on this one... 😅

Reply:
Done https://github.com/SAR-ARD/S1_NRB/pull/123/commits/f2fa33c013f5b967fd86bcb0adca20f225cd8aaf